### PR TITLE
[PR] Capture deadlines, requirements, and contact information

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: includes/class-wsuwp-graduate-degree-programs.php [hunk 1]
```diff
@@ -75,6 +75,16 @@ class WSUWP_Graduate_Degree_Programs {
...
 		),
+		'gsdp_deadlines' => array(
+			'description' => 'Deadlines',
+			'type' => 'deadlines',
+			'sanitize_callback' => 'WSUWP_Graduate_Degree_Programs::sanitize_deadlines',
+		),
+		'gsdp_requirements' => array(
+			'description' => 'Requirements',
+			'type' => 'requirements',
+			'sanitize_callback' => 'WSUWP_Graduate_Degree_Programs::sanitize_requirements',
```
Comment (philcable) [inline includes/class-wsuwp-graduate-degree-programs.php:86]: There's no accompanying `sanitize_requirements` function :). I'm guessing you'll add one later or reuse `sanitize_deadlines`, depending on the data these fields are meant to collect.

**Review by philcable — APPROVED** [review]
👍  Looks good! Left one comment, so I'll leave it up to you whether to merge or address now.

File: includes/class-wsuwp-graduate-degree-programs.php (hunk 1 above)
Comment (jeremyfelt) [reply includes/class-wsuwp-graduate-degree-programs.php:86]: Great catch! I found this at the same time you were reviewing and added it in #12. Thanks :)

**Pull request merged**